src/main/scala/no/ndla/network/NdlaClient.scala

```diff
@@ -55,7 +56,10 @@ trait NdlaClient {
     private def parseResponse[A](response: HttpResponse[String])(implicit mf: Manifest[A]): Try[A] = {
       Try(parse(response.body).camelizeKeys.extract[A]) match {
         case Success(extracted) => Success(extracted)
-        case Failure(ex) => Failure(new HttpRequestException(s"Could not parse response ${response.body}", Some(response)))
+        case Failure(ex) => {
+          logger.warn("Could not parse response", ex)
```

Comment:
Siden dette logges i et bibliotek er det kanskje ryddig å legge med den informasjonen? Vil log-konfigurasjonen for den enkelte komponenten som bruker networks bli brukt her?